test/test_transformers.py

```diff
@@ -1448,6 +1448,18 @@ def test_invalid_last_dim_stride(self, device, kernel: SDPBackend):
                 self.assertRaises(RuntimeError, lambda: torch.nn.functional.scaled_dot_product_attention(
                     q, k, v, None, 0.0, False))
 
+    @onlyCUDA
+    @skipIfRocm  # Nested Tensor
+    @unittest.skipIf(not PLATFORM_SUPPORTS_FLASH_ATTENTION, "Does not support SDPA or pre-SM80 hardware")
+    def test_invalid_mem_efficient_kernel_grouped_query_attention(self, device, ):
```

Comment:
now that cudnn has been added can we also update this test to make sure that it raises

Reply:
we should also add a test that checks the cpu flash throws for gqa

Reply:
You will likely need to rebase this PR in order to see the 'cudnn' updates

Reply:
cudnn doesn't call check_batch_size_and_num_heads_dense(). Unless this function is called gqa check won't be called, and the existing process will work.